### ISSUE TITLE
error when trying to open a file via dropbox

### ISSUE THREAD

**Issue by theoryshaw** [benweet/stackedit#1120]
Getting the following error when trying to open a file via dropbox...

`Dropbox error (400: {"error": "v1_retired"}).`

**Comment by theoryshaw** [comment]
> API v1 retirement
> 
> In June 2016, we announced the deprecation timeline for API v1. When API v1 is retired in September 2017, any further API v1 calls will fail with a 400 error with the body:
> {"error": "v1_retired"}

from: https://www.dropbox.com/developers/reference/migration-guide

**Comment by benweet** [comment]
Can you try with the new v5 please ? (https://stackedit.io/app)

**Comment by theoryshaw** [comment]
yes, worked.  PDF exporter, unfortunately is not working yet in 'app'.  I assume you're aware.

**Comment by RoyiAvital** [comment]
I have issues opening file from Dropbox.
For instant I synchronized with my Dropbox account the Welcome File (`Welcome file.md`).

I authorized StackEdit "Restrict" access (Namely access StackEdit folder only).

When I tried opening it again in a new session I got this:

![image](https://user-images.githubusercontent.com/7347975/36234087-d47c22e0-11f1-11e8-906e-6eb9ccd58022.png)

On top of that a small suggestion, why the name of StackEdit folder has the `(restricted)` in it?
Couldn't it be StackEdit and that's it?

Thank You.
